pkg/clusterconfig/clusterconfig.go

```diff
@@ -25,6 +30,8 @@ const (
 	installerConfigNamespace = "kube-system"
 	installerConfigName      = "cluster-config-v1"
 	installerAWSCredsName    = "aws-creds"
+	imageRegistryNamespace   = "openshift-image-registry"
+	userPrivateConfigName    = "image-registry-private-configuration-user"
```

Comment:
where's the `image-registry-private-configuration` constant defined?  can be put this with that or vice versa?

Reply:
It wasn't, but i added it in the next push